pkg/lwip/include/lwip.h

```diff
@@ -7,13 +7,7 @@
  */
 
 /**
- * @defgroup    pkg_lwip    lwIP
- * @ingroup     pkg
```

Comment:
This is removed without replacement, causing lwIP not to show up under `pkg` anymore.

Reply:
This is already the case in master (and it's related to #8664).

Reply:
You are right